Tangents aren't being written, causing a crash

This is a tracking issue, PR incoming

```
[fred@laptop MFEKglif]$ make testrun
RUSTFLAGS=-Awarnings RUST_LOG=debug RUST_BACKTRACE=1 cargo run -- Q_.glif
   Compiling MFEKMath v0.1.0 (/home/fred/Workspace/math)
   Compiling MFEKglif v0.1.0 (/home/fred/Workspace/MFEKglif)
    Finished dev [unoptimized + debuginfo] target(s) in 7.21s
     Running `target/debug/MFEKglif Q_.glif`
[2021-03-15T03:02:44Z DEBUG MFEKglif::io] Loaded "Q" (U+0051) from Q_.glif

VWSHandle missing tangent

Requested backtrace:
   0: MFEKglif::util::set_panic_hook::{{closure}}
             at src/util/mod.rs:42:26
   1: std::panicking::rust_panic_with_hook
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/panicking.rs:581:17
   2: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/panicking.rs:484:9
   3: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/sys_common/backtrace.rs:153:18
   4: rust_begin_unwind
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/panicking.rs:483:5
   5: core::panicking::panic_fmt
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/core/src/panicking.rs:85:14
   6: core::option::expect_failed
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/core/src/option.rs:1229:5
   7: core::option::Option<T>::expect
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/core/src/option.rs:349:21
   8: MFEKMath::variable_width_stroking::parse_vws_lib
             at /home/fred/Workspace/math/src/variable_width_stroking.rs:377:36
   9: MFEKglif::events::vws::on_load_glif::{{closure}}
             at src/events/vws.rs:211:37
  10: std::thread::local::LocalKey<T>::try_with
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/thread/local.rs:272:16
  11: std::thread::local::LocalKey<T>::with
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/thread/local.rs:248:9
  12: MFEKglif::events::vws::on_load_glif
             at src/events/vws.rs:207:5
  13: MFEKglif::main
             at src/main.rs:92:5
  14: core::ops::function::FnOnce::call_once
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/core/src/ops/function.rs:227:5
  15: std::sys_common::backtrace::__rust_begin_short_backtrace
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/sys_common/backtrace.rs:137:18
  16: std::rt::lang_start::{{closure}}
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/rt.rs:66:18
  17: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/core/src/ops/function.rs:259:13
      std::panicking::try::do_call
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/panicking.rs:381:40
      std::panicking::try
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/panicking.rs:345:19
      std::panic::catch_unwind
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/panic.rs:382:14
      std::rt::lang_start_internal
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/rt.rs:51:25
  18: std::rt::lang_start
             at /rustc/b1af43bc63bc7417938df056f7f25d456cc11b0e/library/std/src/rt.rs:65:5
  19: main
  20: __libc_start_main
  21: _start

make: *** [Makefile:6: testrun] Error 101
```

--- a/src/variable_width_stroking.rs
+++ b/src/variable_width_stroking.rs
@@ -445,6 +445,7 @@ pub fn generate_vws_lib(vwscontours:  &Vec<VWSContour>) -> Option<xmltree::Eleme
             let mut handle_node = xmltree::Element::new("handle");
             handle_node.attributes.insert("left".to_owned(), handle.left_offset.to_string());
             handle_node.attributes.insert("right".to_owned(), handle.right_offset.to_string());
+            handle_node.attributes.insert("tangent".to_owned(), handle.tangent_offset.to_string());
 
 
             match handle.interpolation {

